catalog/spatial/geocoding/spatial-geocoding-create/src/test/java/org/codice/ddf/spatial/geocoding/create/TestGeoNamesCreator.java

```diff
@@ -33,7 +33,8 @@ public void testNoEmptyFields() {
                 -111.84125,
                 "PPL",
                 236123,
-                "Candler,Candleris,Chandler,Chandlur");
+                "Candler,Candleris,Chandler,Chandlur",
+                "US");
```

Comment:
The GeoNames webservice and the Local index both store ISO 3166-1 2 letter format. I believe this is the most widely used format. Is there some sort of government standard on this I can refer to, or do we generally prefer 3 letter format?